backend-question-service/src/routes/questionsRoute.ts

```diff
@@ -0,0 +1,151 @@
+import { Router, Request, Response } from 'express'
+import { Question } from '../models/questionModel'
+
+const router: Router = Router()
+
+// retrieves entire list of questions
+router.get('/', async (req: Request, res: Response) => {
+    try {
+        const questions = await Question.find({})
```

Comment:
question (from someone who hasn't touched express): are the questions sorted in any particular order or are you expecting frontend to sort it? from my django experience i always had to sort the model list in the backend before i transmit the data to the frontend since it's more deterministic. but just curious!

Reply:
I can sort the questions based on any of its attribute, e.g. date it is created, title in alphabetical order etc. Example code below

```
const questions = await Question.find({}).sort({ createdAt: 1 })
```
or
```
const questions = await Question.find({}).sort({ title: 1 })
```